com.raggesilver.BlackBox.json

```diff
@@ -0,0 +1,78 @@
+{
+  "app-id" : "com.raggesilver.BlackBox",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "42",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "blackbox",
+  "finish-args" : [
+    "--allow=devel",
```

Comment:
why do you need devel permissions?

Reply:
The `flatpak-spawn` call Black Box makes needs `--host`, which requires `--allow=devel`.

Reply:
You don't need --allow-devel to use flatpak-spawn --host. You only need --talk-name=org.freedesktop.Flatpak.